app/Keycloak/Service/ClientToKeycloakConverter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Service;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\IntegrationPartnerStatus;
+use Ramsey\Uuid\UuidInterface;
+
+final class ClientToKeycloakConverter
```

Comment:
Is this not an integration to client converter? It seems to convert an integration to a client for creation inside Keycloack

Reply:
Good point, changed the name